backend/test_transport_factory.py

```
@@ -0,0 +1,117 @@
+        verify_ssl=False,
+        timeout=30
+    )
+    
+    # Create transport
+    transport = MCPTransportFactory.create_transport(
+        server_id="test-http-server",
+        transport=http_config
+    )
+    
+    print(f"✅ Created HTTP transport: {type(transport).__name__}")
+    print(f"   - Server ID: test-http-server")
```

Ruff (0.13.1)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)